kudo-operator/operator/templates/spark-applications-crds.yaml

```diff
@@ -5087,3 +5088,4 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
+{{- end }}
```

Comment:
pls add a newline

Reply:
it was there

Reply:
It's strange. Locally I see the new line.

Reply:
GH says differently 😉